AISKU/Tests/Unit/src/CdnPackaging.tests.ts

```diff
@@ -133,38 +137,131 @@ export class CdnPackagingChecks extends AITestClass {
             // Remove any "exports"
             exports = {};
 
-            // Used to simulate loading a cjs module
-            let cjsModule = this["_cjsModule"] = {};
+            // Used to simulate globals without overriding them
```

Comment:
Fixing the tests that where broken by the qunit and requirejs updates. But this also provided the ability to more fully tests the different initialization options